src/lib/helpers/auth.test.ts

```
@@ -0,0 +1,85 @@
+
+// Simple auth helper functions that don't require external dependencies
+export const sessionCookieName = 'sashakt-session';
+
+export function isValidEmail(email: string): boolean {
+	const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+	return emailRegex.test(email);
+}
+
+export function generateSessionId(): string {
+	return Math.random().toString(36).substring(2) + Date.now().toString(36);
+}
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)